xml/obs_request_and_review_system.xml

```diff
@@ -220,4 +220,16 @@
    </itemizedlist>
   </sect2>
  </sect1>
+ <sect1>
+  <title>Who can accept a request</title>
+  <para>
+    The question of who can accept a request can be answered by inspecting who
+    has write permissions on the target project of the request. This includes
+    both maintainers listed in the project itself and all maintainers of higher-
+    level projects within the project hierarchy. (For example: the set of
+    maintainers of a project called "foo:bar" is equal to the union of two
+    sets: the set of project maintainers of the foo:bar project and that of the
+    parent project, foo, itself.)
```

Comment:
```suggestion
    level projects within the project hierarchy.
```

Reply:
A union of two sets is too engineering for my taste. I think you basically repeat inside the brackets what you explained before in more difficult words...

Reply:
The OBS is a tool for engineers, who are supposed to have been exposed to basic maths, but I get your point!